modules/ducktests/tests/ignitetest/tests/add_node_rebalance_test.py

```diff
@@ -39,20 +39,17 @@ class AddNodeRebalanceTest(IgniteTest):
     REBALANCE_TIMEOUT = 60
 
     @cluster(num_nodes=NUM_NODES + 1)
-    @parametrize(version=str(DEV_BRANCH))
-    @parametrize(version=str(LATEST))
-    def test_add_node(self, version):
+    @ignite_versions(str(DEV_BRANCH), str(LATEST))
```

Comment:
As we introduce own decorator to work with versions - let's remove requirement to use str for versions. It's ugly

Reply:
Unfortunatelly, args should be json serializable. I'm afraid that it is not possible.

Reply:
It is possible after one bug in ducktape is resolved. I'll fix it ASAP, then I will rewrite it